dfastmi/batch.py

```diff
@@ -1218,7 +1225,7 @@ def analyse_and_report_dflowfm(
         interest_region[iface] = 1
     
         #if display:
-        #    dfastmi.io.log_text('-- get centres') # note that this should be the circumference point
+        #    ApplicationSettingsHelper.log_text('-- get centres') # note that this should be the circumference point
```

Comment:
maybe we should discuss with Bert if this can be removed during the batch refactoring

Reply:
yes i agree